fightScreen.py

```diff
@@ -0,0 +1,649 @@
+                QSpacerItem(0, 10, QSizePolicy.Fixed, QSizePolicy.Fixed), arsRow, 1
+            )
+            arsRow += 1
+
+            self.P1_armor1_Lbl2 = QLabel("")
+            self.P1_armor1_Lbl2.setAlignment(Qt.AlignCenter)
+            self.P1_armor1_Lbl2.setPixmap(self.imgDict[key].scaledToWidth(80))
+            self.P1defenseLayout.addWidget(self.P1_armor1_Lbl2, arsRow, 1)
+            arsRow += 1
+
+            self.P1defenseLayout.addItem(
+                QSpacerItem(0, 10, QSizePolicy.Fixed, QSizePolicy.Fixed), arsRow, 1
+            )
+            arsRow += 1
+
+            self.P1_armor1_Lbl3 = QLabel(
+                "H: "
+                + str(self.P1DefenseArray[key][0])
+                + "\nM: "
+                + str(self.P1DefenseArray[key][1])
+                + "\nMech: "
+                + str(self.P1DefenseArray[key][2])
+            )
+            self.P1_armor1_Lbl3.setAlignment(Qt.AlignCenter)
+            self.P1_armor1_Lbl3.setStyleSheet("color: white;")
+            self.P1defenseLayout.addWidget(self.P1_armor1_Lbl3, arsRow, 1)
+            arsRow += 1
+
+            self.P1defenseLayout.addItem(
+                QSpacerItem(0, 40, QSizePolicy.Fixed, QSizePolicy.Fixed), arsRow, 1
+            )
+            arsRow += 1
+        self.P1defenseLayout.addItem(
+            QSpacerItem(0, 40, QSizePolicy.Fixed, QSizePolicy.MinimumExpanding),
+            arsRow,
+            1,
+        )
+
+        #############################################################
+
+        # self.P1defenseArsenal = Arsenal('Defense', self.P1DefenseArray)
+        self.mainLayout.addWidget(self.P1defenseWgt, row, colm, 16, 1)
+        colm += 1
+
+        self.mainLayout.addItem(
+            QSpacerItem(
+                40, 40, QSizePolicy.MinimumExpanding, QSizePolicy.MinimumExpanding
+            ),
+            row,
+            colm,
+        )
+        row += 1
+        colm += 1
+
+        innerCol = colm
+        rightCol = colm + 4
+        self.player1_Lbl = QLabel("Player 1")
+        self.player1_Lbl.setStyleSheet("color: white; font-size: 30px;")
+        self.mainLayout.addWidget(self.player1_Lbl, row, innerCol, 1, 3)
+
+        self.player2_Lbl = QLabel("Player 2")
+        self.player2_Lbl.setStyleSheet("color: white; font-size: 30px;")
+        self.mainLayout.addWidget(self.player2_Lbl, row, rightCol, 1, 3)
+        row += 1
+
+        self.mainLayout.addItem(
+            QSpacerItem(0, 30, QSizePolicy.Fixed, QSizePolicy.Fixed), row, 1
+        )
+        row += 1
+
+        self.player1_meters = {
+            "health": Meter(QColor(255, 0, 0), 100),
+            "mechanical": Meter(QColor(50, 50, 50), 100),
+            "magic": Meter(QColor(200, 0, 200), 100),
+        }
+
+        self.player2_meters = {
+            "health": Meter(QColor(255, 0, 0), 100),
+            "mechanical": Meter(QColor(50, 50, 50), 100),
+            "magic": Meter(QColor(200, 0, 200), 100),
+        }
```

Comment:
What is the point of having these and also
```python
        self.P1HealthMeter = 100
        self.P2HealthMeter = 100
        self.P1MagicMeter = 100
        self.P2MagicMeter = 100
        self.P1MechMeter = 100
        self.P2MechMeter = 100
```
?

Reply:
Especially considering they're not being used in the `Meter`, which takes in a color and percentage.
```python
red = QColor(255, 0, 0)
Meter(red, 50)  # Red meter 50% filled
```

Reply:
This was just for visual, functionality should be an assigned issue after merge.